src/main/java/seedu/contax/logic/commands/RangeCommand.java

```diff
@@ -0,0 +1,114 @@
+package seedu.contax.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.contax.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_RANGE_FROM;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_RANGE_TO;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.contax.commons.core.Messages;
+import seedu.contax.commons.core.index.Index;
+import seedu.contax.logic.commands.exceptions.CommandException;
+import seedu.contax.logic.parser.AddressBookParser;
+import seedu.contax.logic.parser.ArgumentMultimap;
+import seedu.contax.logic.parser.exceptions.ParseException;
+import seedu.contax.model.Model;
+
+/**
+ * Range edit or delete a person identified using it's displayed from index and to index from the address book.
+ */
+public class RangeCommand extends Command {
+    public static final String COMMAND_WORD = "range";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": command in range"
+            + "by the index number used in the displayed person list. "
+            + "Parameters: from/FROM_INDEX to/TO_INDEX (must be a positive integer) "
+            + "Example command range edit and range delete ";
```

Comment:
Please follow the convention set out by the rest of the commands for the example command.

Reply:
thx and modified